apps/examples/sensorbd_demo/examples/spi_k6ds3.c

```diff
@@ -163,9 +163,8 @@ void k6ds3_main(int argc, char *argv[])
 		gy = (gyl | (gyh << 8));
 		gz = (gzl | (gzh << 8));
 
-		printf("ACCEL:x(0x%04X) y(0x%04X) z(0x%04X) GYRO:x(0x%04X) y(0x%04X) z(0x%04X)\n",
-			ax, ay, az, gx, gy, gz);
+		printf("ACCEL:x(0x%04X) y(0x%04X) z(0x%04X) GYRO:x(0x%04X) y(0x%04X) z(0x%04X)\n", ax, ay, az, gx, gy, gz);
```

Comment:
TizenRT coding standard explicitly requires this? IMO, putting a new line to stay within 80 columns is no problem at all. We intentionally do it.